### PR TITLE
Publishing the frontend

### PR DESCRIPTION
**Pull request by joostdebruijn**
The original front-end implementation for this project.

### CODE REVIEW

File: frontend/dashboard/getProducts [hunk 1]
```diff
@@ -0,0 +1,183 @@
+[
```
Comment (FlorisHoogenboom) [inline frontend/dashboard/getProducts]: I think these statistics should be fetched from the endpoint GET /getProducts.

Comment (joostdebruijn) [reply frontend/dashboard/getProducts]: Already happens, see [centre.js:59](https://github.com/joostdebruijn/O-T-P/blob/d2a81d6d2bc76efe38e4362601a0994ba87b1ad7/frontend/dashboard/centre.js#L59). Should I remove the sample file?

Comment (FlorisHoogenboom) [reply frontend/dashboard/getProducts]: Please! I'll generate a sample config in the backend to replace the now encrypted config file.

**Comment by joostdebruijn** [comment]
Done!

**Pull request merged**